examples/bow/Ballerina.toml

```diff
@@ -2,4 +2,4 @@
 distribution = "2201.0.0-20211221-194900-b414ea8e"
 
 [build-options]
-observabilityIncluded = true
+observabilityIncluded = false
```

Comment:
Any reason to turn it off ?

Reply:
Here's the reason https://github.com/ballerina-platform/module-ballerina-io/pull/458 @LakshanKarunathilake could you please give more insight into this?

Reply:
By this level `module-ballerinai-observe` is not built, this is level 1 module and `module-ballerinai-observe` is in level 2. The same reason we made `observabilityIncluded` flag false in ballerina-lang repo test-cases.

Reply:
The ideal solution would be to build observe modules before standard library build. So we don't need to disable it. Let's merge it for now